be/src/exec/pipeline/exchange/local_exchange_memory_manager.h

```diff
@@ -42,9 +44,14 @@ class LocalExchangeMemoryManager {
 
     bool is_full() const { return _memory_usage >= _max_memory_usage; }
```

Comment:
we should add a DCHECK for local_exchange DCHECK(!is_full() || has_output());

Reply:
> we should add a DCHECK for local_exchange DCHECK(!is_full() || has_output());

what if no data input, I think this check is not reasonable...